### PR TITLE
Add Mx Power Gadget 1.2 to cask

### PR DESCRIPTION
**Pull request by mavethee**
I've noticed there is AMD and Intel Power Gadgets available but the one for M1s is missing so here it is

### CODE REVIEW

**Comment by bevanjkay** [comment]
Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

**Comment by mavethee** [comment]
> Please try to add a [zap stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

Done, it's seems to share same kinds of files like Intel Power Gadget, should be fine now, no idea what to do with failing audit.

**Comment by bevanjkay** [comment]
@mavethee run `brew style --fix mx-power-gadget`
and also note from CI - 
```
Use `sha256 :no_check` when URL is unversioned.
```

File: Casks/mx-power-gadget.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+cask "mx-power-gadget" do
+    version "1.2,20220322"
+    sha256 "702f6ee1e533334bd6ffa42c168b20ad4d39a4c9a197fd0ece7e4b196bfdb929"
```
Comment (bevanjkay) [inline Casks/mx-power-gadget.rb:3]: @mavethee Because the `url` is unversioned we cannot provide a shasum.
```suggestion
  sha256 :no_check
```

**Review by bevanjkay — CHANGES_REQUESTED** [review]
Please also run `brew style --fix mx-power-gadget` again also

**Review by bevanjkay — DISMISSED** [review]

**Comment by mavethee** [comment]
Any ideas on that cask failing for ubuntu? if that really matters?

**Comment by bevanjkay** [comment]
@mavethee It is failing the style checks - running `brew style --fix mx-power-gadget` and committing the changes should clear it up.

**Comment by mavethee** [comment]
> brew style --fix mx-power-gadget

Well I did, no offenses detected

**Comment by mavethee** [comment]
Thanks!

**Review by bevanjkay — APPROVED** [review]

**Pull request merged**